org.eclipse.jdt.ls.product/languageServer.product

```diff
@@ -40,12 +41,14 @@
       <plugin id="org.eclipse.debug.core"/>
       <plugin id="org.eclipse.equinox.app"/>
       <plugin id="org.eclipse.equinox.common"/>
+      <plugin id="org.eclipse.equinox.ds"/>
```

Comment:
which plugin introduces the dependencies for org.apache.felix.scr and org.eclipse.equinox.ds?

Reply:
There aren't any dependencies, but Java LS will not start without them. See also https://bugs.eclipse.org/bugs/show_bug.cgi?id=538737

Reply:
so if org.eclipse.equinox.ds is deprecated, would you still want to keep it in languageServer.product? 

Also we need verify whether to update syntaxServer.product file either. In theory, if the new dependencies are introduced by the fundamental platform, they need to be updated for syntaxServer.

Reply:
Fixed.